starter-code/HeterogeneousRecyclerView/app/src/main/java/com/charlesdrews/heterogeneousrecyclerview/FootballRecyclerViewAdapter.java

```diff
@@ -17,6 +17,62 @@
 
 public class FootballRecyclerViewAdapter extends RecyclerView.Adapter {
 
+    List<BaseFootballObject> mObjectList;
+
+    public FootballRecyclerViewAdapter(List<BaseFootballObject> objectList) {
+        mObjectList = objectList;
+    }
+
+    @Override
+    public int getItemViewType(int position) {
+        if(mObjectList.get(position) instanceof Team){
+            return 2;
+        } else if(mObjectList.get(position) instanceof Player){
+            return 1;
```

Comment:
I recommend using constants like PLAYER and TEAM rather than plain int values. This will make your code more readable.